web/skins/classic/js/skin.js

```diff
@@ -346,7 +361,8 @@ if ( currentView != 'none' && currentView != 'login' ) {
     // Manage the web console filter bar minimize chevron
     $j("#mfbflip").click(function() {
       $j("#mfbpanel").slideToggle("slow", function() {
-        changeScale();
+        //changeScale(); //This function disappeared somewhere... It needs to be restored. Function call appeared Aug 10, 2023 https://github.com/ZoneMinder/zoneminder/commit/b8982dce53137d86d33d2d6d79104d4a25673ef7
+        redrawScreen(); //Temporarily use changeScale() instead. You can try using refreshWindow()
```

Comment:
This is a temporary solution.
@connortechnology , maybe you know what happened with the changeScale() function?

Reply:
I think it exists in an unfinished branch of mine.  It exists in watch.js and montage.js.  So we should test for existence of function and use what exists.

Reply:
I added a function exists check